2022-11-03 23:15:57,249 INFO    MainThread:30920 [wandb_setup.py:_flush():75] Loading settings from C:\Users\Tobias\.config\wandb\settings
2022-11-03 23:15:57,249 INFO    MainThread:30920 [wandb_setup.py:_flush():75] Loading settings from C:\Users\Tobias\Desktop\Development\Python\TicTacToe\wandb\settings
2022-11-03 23:15:57,249 INFO    MainThread:30920 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-11-03 23:15:57,249 WARNING MainThread:30920 [wandb_setup.py:_flush():75] Could not find program at -m src.train_ai
2022-11-03 23:15:57,249 INFO    MainThread:30920 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': None, 'program': '-m src.train_ai'}
2022-11-03 23:15:57,250 INFO    MainThread:30920 [wandb_init.py:_log_setup():405] Logging user logs to C:\Users\Tobias\Desktop\Development\Python\TicTacToe\wandb\run-20221103_231557-h465zmzw\logs\debug.log
2022-11-03 23:15:57,250 INFO    MainThread:30920 [wandb_init.py:_log_setup():406] Logging internal logs to C:\Users\Tobias\Desktop\Development\Python\TicTacToe\wandb\run-20221103_231557-h465zmzw\logs\debug-internal.log
2022-11-03 23:15:57,250 INFO    MainThread:30920 [wandb_init.py:init():439] calling init triggers
2022-11-03 23:15:57,250 INFO    MainThread:30920 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-11-03 23:15:57,250 INFO    MainThread:30920 [wandb_init.py:init():492] starting backend
2022-11-03 23:15:57,250 INFO    MainThread:30920 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-11-03 23:15:57,255 INFO    MainThread:30920 [backend.py:ensure_launched():219] starting backend process...
2022-11-03 23:15:57,311 INFO    MainThread:30920 [backend.py:ensure_launched():224] started backend process with pid: 22296
2022-11-03 23:15:57,312 INFO    MainThread:30920 [wandb_init.py:init():501] backend started and connected
2022-11-03 23:15:57,316 INFO    MainThread:30920 [wandb_init.py:init():565] updated telemetry
2022-11-03 23:15:57,363 INFO    MainThread:30920 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-11-03 23:16:01,028 INFO    MainThread:30920 [wandb_init.py:init():619] run resumed
2022-11-03 23:16:01,035 INFO    MainThread:30920 [wandb_run.py:_on_init():1759] communicating current version
2022-11-03 23:16:01,269 INFO    MainThread:30920 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.13.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-11-03 23:16:01,269 INFO    MainThread:30920 [wandb_init.py:init():625] starting run threads in backend
2022-11-03 23:16:01,297 INFO    MainThread:30920 [wandb_run.py:_console_start():1733] atexit reg
2022-11-03 23:16:01,298 INFO    MainThread:30920 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-11-03 23:16:01,298 INFO    MainThread:30920 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-11-03 23:16:01,299 INFO    MainThread:30920 [wandb_run.py:_redirect():1667] Redirects installed.
2022-11-03 23:16:01,299 INFO    MainThread:30920 [wandb_init.py:init():664] run started, returning control to user process
2022-11-03 23:17:07,176 INFO    MainThread:30920 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 1
2022-11-03 23:17:07,177 INFO    MainThread:30920 [wandb_run.py:_restore():1674] restore
2022-11-03 23:17:07,710 INFO    MainThread:30920 [wandb_run.py:_restore():1674] restore
